tests/test_resources.py

```diff
@@ -80,3 +80,43 @@ def test_get_results(self, reqmock):
         assert req_json['remove_layer_view_ids'][0]['ref_id'] == 'yyy'
 
         assert pv.id == 'a1'
+
+    # ARE-6130 wrapper for the exchange rate table unique currencies function
+    def test_unique_currencies(self, reqmock):
+        # mock for the Exchange Rate table request
+        reqmock.get('https://api/exchange_rate_tables/abc_id',
+                    status_code=200, text='{"id":"abc_id"}')
+
+        fx_table = ExchangeRateTable.retrieve('abc_id')
+        # mock for the currencies method call
+        reqmock.get('https://api/exchange_rate_tables/abc_id/currencies', status_code=200,
+                    text='{'
```

Comment:
Can you put the text part in one line? feels that will look a bit nicer.

Reply:
Updated.